### PR TITLE
[Issue 5614] Deprecate brokerServicePurgeInactiveFrequencyInSeconds

### LINKED ISSUE #5614: Deprecate `brokerDeleteInactiveTopicsFrequencySeconds` or `brokerServicePurgeInactiveFrequencyInSeconds`
**Describe the bug**
There are two similar parameters: `brokerDeleteInactiveTopicsFrequencySeconds` and `brokerServicePurgeInactiveFrequencyInSeconds`. 
In documentation, `brokerDeleteInactiveTopicsFrequencySeconds` is recommended. However, `brokerServicePurgeInactiveFrequencyInSeconds` is implemented in code. 

image

**Additional context**
We need deprecate one of them, make it consistent in code and documentation.

### PR DESCRIPTION
**Pull request by lsy180829**
### Motivation

Fixes #5614 

*Explain here the context, and why you're making that change. What is the problem you're trying to solve.*

### Modifications

Deprecate brokerServicePurgeInactiveFrequencyInSeconds.
Modify many files, includes .java, .conf, .md.

### CODE REVIEW

File: pulsar-broker-common/src/main/java/org/apache/pulsar/broker/ServiceConfiguration.java [hunk 1]
```diff
@@ -253,7 +253,7 @@
         category = CATEGORY_POLICIES,
         doc = "How often to check for inactive topics"
     )
-    private long brokerDeleteInactiveTopicsFrequencySeconds = 60;
+    private int brokerDeleteInactiveTopicsFrequencySeconds = 60;
```
[the remark this answers is not in the document]
Comment (lsy180829) [reply pulsar-broker-common/src/main/java/org/apache/pulsar/broker/ServiceConfiguration.java]: https://github.com/apache/pulsar/blob/master/pulsar-broker/src/main/java/org/apache/pulsar/broker/service/BrokerService.java#L406,L410
Cause of these lines, if I use 'long' , there will be an incompatible type. 
Or, change the related method.

File: pulsar-broker-common/src/main/java/org/apache/pulsar/broker/ServiceConfiguration.java [hunk 2]
```diff
@@ -1200,6 +1200,7 @@
         doc = "How often broker checks for inactive topics to be deleted (topics with no subscriptions and no one connected)"
     )
     private int brokerServicePurgeInactiveFrequencyInSeconds = 60;
+    @Deprecated
```
[the remark this answers is not in the document]
Comment (lsy180829) [reply pulsar-broker-common/src/main/java/org/apache/pulsar/broker/ServiceConfiguration.java]: Thank you very much, I'll move it.

**Comment by sijie** [comment]
run java8 tests

**Comment by Jennifer88huang** [comment]
run java8 tests

**Comment by lsy180829** [comment]
run java8 tests

**Comment by sijie** [comment]
run java8 tests

**Comment by sijie** [comment]
run java8 tests

**Pull request merged**